packages/opencensus-instrumentation-grpc/src/grpc-stats/client-metrics.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AggregationType, globalStats, MeasureUnit, View} from '@opencensus/core';
+import {DEFAULT_BYTES_DISTRIBUTION, DEFAULT_MESSAGE_COUNT_DISTRIBUTION, DEFAULT_MILLI_SECONDS_DISTRIBUTION} from './stats-common';
+
+/**
+ * {@link Measure} for number of messages sent in the RPC.
+ *
+ */
+const GRPC_CLIENT_SENT_MESSAGES_PER_RPC = globalStats.createMeasureInt64(
+    'grpc.io/client/sent_messages_per_rpc', MeasureUnit.UNIT,
+    'Number of messages sent in the RPC (always 1 for non-streaming RPCs).');
+
+/**
+ * {@link Measure} for total bytes sent across all request messages per RPC.
+ *
+ */
+const GRPC_CLIENT_SENT_BYTES_PER_RPC = globalStats.createMeasureInt64(
+    'grpc.io/client/sent_bytes_per_rpc', MeasureUnit.BYTE,
+    'Total bytes sent across all request messages per RPC.');
+
+/**
+ * {@link Measure} for number of response messages received per RPC.
+ *
+ */
+const GRPC_CLIENT_RECEIVED_MESSAGES_PER_RPC = globalStats.createMeasureInt64(
+    'grpc.io/client/received_messages_per_rpc', MeasureUnit.UNIT,
+    'Number of response messages received per RPC (always 1 for non-streaming RPCs).');
+
+/**
+ * {@link Measure} for gRPC client uncompressed response bytes.
```

Comment:
This should be " for total bytes received across all response messages per RPC."